docs/concepts/MSBuild-props-and-targets.md

```diff
@@ -0,0 +1,74 @@
+---
+title: MSBuild props and targets in a package
+description: Describes MSBuild props and targets in NuGet packages
+author: nkolev92
+ms.author: nikolev
+ms.date: 07/13/2022
+ms.topic: conceptual
+---
+
+# MSBuild props and targets in a package
+
+In additional to the more traditional assemblies, NuGet packages may sometimes add custom build targets or properties to projects that consume that package.
+This can be achieved by adding a valid MSBuild file, in the form `<package_id>.targets` or `<package_id>.props` (such as `Contoso.Utility.UsefulStuff.targets`) within the build folders of the project.
+
+## Build folders
+
+As NuGet has evolved, various different folders for build props and targets have been added.
+
+| Folder | NuGet Version | Use |

```

Comment:
Great table, how does one differentiate when to use each of these? Should we talk about that outside the use?

Reply:
I kind of assumed the `use` explains that.

Reply:
Like from the developer perspective which is most common? Which one should I use in my situation? I can piece together the answers but maybe we should have another column of common cases?

Reply:
Do you have a suggestion?

I kind of see those 2 as equivalent to the current descriptions.